.storybook/decorators/sizing.js

```diff
@@ -2,56 +2,74 @@ import { makeDecorator } from "@storybook/preview-api";
 import { html } from "lit";
 import { styleMap } from "lit/directives/style-map.js";
 
+import { Template as Typography } from "@spectrum-css/typography/stories/template.js";
```

Comment:
I think we should deprecate the sizing decorator for now and move toward using the Sizes utility. In the future, for docs pages, I think we could make use of a custom doc block for Sizes that will give us a very consistent UX for docs pages.

Reply:
The `Sizes()` utility is now used in the latest rebased work, instead of the sizing decorator.